exporter/mezmoexporter/README.md

```diff
@@ -0,0 +1,30 @@
+# Mezmo Exporter
+
+This exporter supports sending OpenTelemetry log data to [LogDNA (Mezmo)](https://logdna.com).
+
+# Configuration options:
+
+- `ingest_url` (required): Use the default URL of `https://logs.logdna.com/log/ingest` unless a custom URL has been provided for your account.
```

Comment:
If there's a default, why have it as a required field?

Reply:
Modified to allow this value as optional with the default URL supplied if omitted.